src/cmr/concepts/draft.js

```diff
@@ -13,16 +15,58 @@ export default class Draft extends Concept {
   constructor(conceptType, headers, requestInfo, params) {
     super(conceptType, headers, requestInfo, params)
 
-    const singularConceptType = conceptType.replace('drafts', 'draft')
     const {
-      providerId,
-      ummVersion
+      draftConceptId = '',
```

Comment:
This feels like a required field, what is the purpose of defaulting it to an empty string versus throwing an error?

Reply:
I think the empty string came from an earlier attempt to work this where I was performing some action on the string, not needed anymore